hw2_decor_iter/HW_2.py

```diff
@@ -0,0 +1,337 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Задание 1 (2 балла)
+
+# Напишите класс `MyDict`, который будет полностью повторять поведение обычного словаря, за исключением того, что при итерации мы должны получать и ключи, и значения.
+# 
+# **Модули использовать нельзя**
+
+# In[22]:
+
+
+class MyDict(dict):
+    def __iter__(self):
+        return zip(self.keys(), self.values())
```

Comment:
Норм, но можно вот так
```suggestion
        return iter(self.items())
```